panel/models/echarts.ts

```diff
@@ -80,7 +80,7 @@ export class EChartsView extends HTMLBoxView {
   _plot(): void {
     if ((window as any).echarts == null)
       return
-    this._chart.setOption(this.model.data);
+    this._chart.setOption(this.model.data, true);
```

Comment:
How do we know

- That it will work across the use cases we need to support?
  - I believe we loose the nice transitions that Echarts provides today. I would recommend having a parameter holding the `true`/ `false` value. `false` is default and provides nice transitions. `true` if you need the workaround for removing series?
- That it  does not have performance implications to rerender from scratch every time?

Reply:
In my experiments Echarts is on top when streaming data compared to bokeh, holoviews, plotly and altair. That we might risk destroying?

Reply:
Thanks for the feedback Marc :) 
I have pushed a new commit.  Now, instead of passing true by default, a dict is passed. It's named `options_kwargs` and is added in the TS model as well as the python model and echarts pane.

So new, we can either decide to leave the behavior as is, or we can do : 
```py 
pn.pane.ECharts(my_data,
        options_kwargs={ "replaceMerge": ['series'] }) # similar to true
```

Solution based on [this comment](https://github.com/apache/echarts/issues/6202#issuecomment-974761211) by a core contributor to Echarts.